solutions/bio1/ParsingLists/index.tsx

```diff
@@ -1,7 +1,84 @@
-import { STitle } from "components";
+import { Link, SCodeBlock, SList, SText, STitle } from "components";
+import { brace, floor } from "solutions/util";
 
 export const ParsingLists = () => {
-  return <>
-    <STitle>Coming Soon!</STitle>
-  </>
+  return (
+    <>
+      <STitle>Observation</STitle>
+      <SText>
+        At no point does any value in the list affect any other value in the list. This means that at each point, instead of building the entire list (which would be far too slow), we can simply calculate the value at the position required.
+      </SText>
+
+      <STitle>Sample Explanation</STitle>
+      <SText>
+        The $i$th element with the description E(OE) is equal to: E[O[E[E[E[O[E[i]]]]]]].
+        Now, working from the inside out:
+      </SText>
+      <SList>
+        <SText>E[3] = 6</SText>
+        <SText>O[6] = 11</SText>
+        <SText>E[11] = 22</SText>
+        <SText>E[22] = 44</SText>
+        <SText>E[44] = 88</SText>
+        <SText>O[88] = 175</SText>
+        <SText>E[175] = 350</SText>
+      </SList>
+      <SText>
+        So the $i$th element with the description E(OE) is equal to 350, when $i=3$.
+      </SText>
+
+      <STitle>Idea</STitle>
+      <SText>
+        This problem is broken down into two parts:
+      </SText>
+      <SList>
+        <SText>
+          Efficiently calculating O[i], E[i] and T[i] for any $i$, however large.
+        </SText>  
+        <SText>
+          Parsing the description given in the input into a list of operations to perform in order.
+        </SText>
+      </SList>
+
+      <STitle>Solution</STitle>
+      <SText>
+        For the first part, instead of treating O, E and T as lists, treat them as functions, where $O(x)$ is the value of O[x], $E(x)$ is the value of E[x] and $T(x)$ is the value of T[x].
+      </SText>
+      <SList>
+        <SText>
+          It is easy to see that $E(x) = 2x$, since E = [2, 4, 6, 8, 10, 12...]
```

Comment:
please use dollar signs around the list.
also, use \dots instead of writing ... in latex mode

so for this you'd write $[2, 4, 6, 8, 10, 12, \dots]$